agave_app/python/RenderInterface.h

```diff
@@ -70,4 +70,9 @@ class RenderInterface
   virtual int BackgroundColor(float, float, float) = 0;
   virtual int SetIsovalueThreshold(int32_t, float, float) = 0;
   virtual int SetControlPoints(int32_t, std::vector<float>) = 0;
+  virtual int SetBoundingBoxColor(float, float, float) = 0;
+  virtual int ShowBoundingBox(int32_t) = 0;
+  virtual int ShowScaleBar(int32_t) = 0;
+  virtual int SetFlipAxis(int32_t, int32_t, int32_t) = 0;
+  virtual int SetInterpolation(int32_t) = 0;
```

Comment:
the above functions were missing, so I just threw them in for consistency. (This class ideally is the complete command api) This code in here is actually dead code currently and probably should be removed in a separate PR.  It was meant as a way to do a cleaner python integration that didn't require you to run agave separately as a server process, but never got it fully working.